README.md

```diff
@@ -0,0 +1,74 @@
+## HERMES GitHub and GitLab Plugin for Metadata Extraction
+> This plugin automatically harvests metadata from GitHub and GitLab repositories.
+> The extraction follows the **[CodeMeta](https://codemeta.github.io/)** standard to ensure compatibility and interoperability with other metadata tools and formats.
```

Comment:
Can you please add an information that we also follow their crosswalk and link that one here?